packages/toolpad-app/src/toolpad/AppEditor/AppAuthorizationEditor.tsx

```diff
@@ -177,6 +182,20 @@ export function AppAuthenticationEditor() {
           placeholder="example.com"
         />
       ))}
+      {!isPaidPlan ? (
+        <UpgradeAlert
```

Comment:
Maybe this alert for authentication providers would look better just below the provider input (below the blue alert)? Anyway I'll approve it already, not a big deal.

Reply:
Ok, that's fine then!